project/Dependencies.scala

```diff
@@ -115,7 +115,12 @@ object Dependencies {
   val Sqs = Seq(
     libraryDependencies ++= Seq(
       "com.amazonaws"     %  "aws-java-sdk-sqs"    % "1.11.109",             // ApacheV2
-      "org.elasticmq"     %% "elasticmq-rest-sqs"  % "0.13.4"        % Test, // ApacheV2
+      "org.elasticmq"     %% "elasticmq-rest-sqs"  % "0.13.4"        % Test excludeAll(
+        // elasticmq-rest-sqs depends on Akka 2.5, exclude it, so we can choose Akka version
+        ExclusionRule(organization = "com.typesafe.akka") //
```

Comment:
The problem was mixing Akka 2.4 and 2.5 versions. Without this exclusion rule and when alpakka is compiled with 2.4, `elasticmq-rest-sqs` would evict akka-stream and akka-actor to 2.5.1 version, but not akka-stream-testkit. Therefore we would end up with newer akka-actor and akka-stream and older akka-stream-testkit which would cause `NoClassDefFoundError` mentioned in the original ticket.

By adding this ExclusingRule, we deny any control over Akka version from this library and then we test with Akka 2.4 and Akka 2.5.